useSelect returns undefined instead of trow an error when mapSelect trows

Hi, i'm trying to make custom gutenberg block with predefined layout selection. So i've used 'core/column' & 'core/columns' as start point.

But now i'm facing a problem, when i try to create a block of 'gutenberg-modules/columns' i'm getting following error in a console:

````
TypeError: Cannot read property 'blockType' of undefined
    at ColumnsEdit (edit.js:214)
    ...
````

So, this function implemented just as original one and contains following code:
````javascript
const {
	blockType,
	defaultVariation,
	hasInnerBlocks,
	variations,
} = useSelect(
	(select) => {
		const {
		__experimentalGetBlockVariations,
		getBlockType,
		__experimentalGetDefaultBlockVariation,
	} = select('core/blocks');

	return {
		blockType: getBlockType(name),
		defaultVariation: __experimentalGetDefaultBlockVariation(
			name,
			'block'
		),
		hasInnerBlocks:
			select('core/block-editor').getBlocks(clientId).length >
			0,
			variations: __experimentalGetBlockVariations(name, 'block'),
		};
	},
	[clientId, name]
);
````
As i mentioned, `(select) => {...}` works just fine and returns correct data, but useSelect returns undefined by itself.

Full code can be found [here](https://github.com/mevepe/gutenberg-modules)

Look like its `__experimentalGetDefaultBlockVariation`, i've removed destruct part and got real arror: `backend.js:6 Error: An error occurred while running 'mapSelect': __experimentalGetDefaultBlockVariation is not a function`

--- a/packages/data/src/components/use-select/index.js
+++ b/packages/data/src/components/use-select/index.js
@@ -114,6 +114,9 @@ export default function useSelect( _mapSelect, deps ) {
 			errorMessage += 'Original stack trace:';
 
 			throw new Error( errorMessage );
+		} else {
+			// eslint-disable-next-line no-console
+			console.error( errorMessage );
 		}
 	}
 

